lib/mix/lib/mix/tasks/run.ex

```diff
@@ -12,7 +12,7 @@ defmodule Mix.Tasks.Run do
       mix run my_script.exs
 
   This task provides a subset of the functionality available in the
-  `elixir` executable, including setting up the `System.argv`:
+  `elixir` executable, including setting up the `System.argv/1` arguments:
```

Comment:
I would use `System.argv/0` here. The rationale is that, although `System.argv/1` is likely what we are using internally, the user will consume those changes via `System.argv/0`.